test/scripts/modify-plain-specs.ts

```diff
@@ -11,7 +11,7 @@ yargs(hideBin(process.argv))
   .usage("Usage: $0")
   .version("2.0.0")
   .command(
-    `process <inputPath> <outputPath>`,
+    "process <inputPath> <outputPath> [runtimePath]",
```

Comment:
We are not parsing any `[runtimePath]` in this script

Reply:
Done